luaui/Widgets/gui_unit_stats.lua

```diff
@@ -596,8 +596,8 @@ local function drawStats(uDefID, uID)
 				uWep = WeaponDefNames[uWep.customParams.def] or uWep
 				defaultDamage = uWep.damages[0]
 			elseif uWep.customParams.cluster then
-				local munition = uWep.customParams.def    or uDef.name .. '_' .. 'cluster_munition'
-				local cmNumber = uWep.customParams.number or 5 -- note: keep in sync with cluster defaults
+				local munition = uDef.name .. '_' .. uWep.customParams.cluster_def
+				local cmNumber = uWep.customParams.cluster_number or 5 -- note: keep in sync with cluster defaults
```

Comment:
What happens if somebody doesn't keep the 5 in sync with cluster defaults (whatever they are)? Why not do `customParams.cluster_number or error("please specify cluster_number")`

Reply:
Unit stats widget is a second class citizen, I missed this. Error for a missing required custom param should be done in preprocessing, not some random widget.

Engine paradigm is to provide default values for missing def tags, so it makes sense to have a default rather than throw an error. I am loathe to go through the trouble of synced-unsynced communication to pass the default value here, might as well just do it in alldefs_post.lua.

Reply:
[Discord thread about added def tags and default values](https://discord.com/channels/549281623154229250/1347249345765969980)

Reply:
Implemented inside alldefs_post.lua